package.json

```diff
@@ -53,7 +53,7 @@
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
     "react-dom": "^15.4.2",
-    "react-scripts": "0.8.5",
+    "react-scripts": "0.9.0",
```

Comment:
This enables source maps as discussed here: https://github.com/facebookincubator/create-react-app/issues/343

![image](https://cloud.githubusercontent.com/assets/155953/22952552/a7f5220c-f2f4-11e6-9a1e-8e9eaf653c88.png)